doc/rst/platforms/omnipath.rst

```diff
@@ -0,0 +1,159 @@
+Most Omni-Path clusters use a workload manager or queueing system
+such a Slurm. To select an appropriate Chapel launcher you can set
+``CHPL_LAUNCHER`` to one of the following values:
+
+===================  ======================================
+Launcher Name        Description
+===================  ======================================
+gasnetrun_ofi         run jobs interactively on your system
+slurm-gasnetrun_ofi   queue jobs using Slurm (srun/sbatch)
+===================  ======================================
+
+.. _setting-ofi-spawner:
+
+-------------------
+Selecting a Spawner
+-------------------
+
+Under the covers ``gasnetrun_ofi`` based launchers must figure out
+how to spawn jobs and get them up and running on the compute nodes.
+GASNet's two primary means of doing this on Omni-Path clusters are
+``ssh`` and ``mpi``. GASNet will default to ``mpi`` if MPI support
+is detected at configure time, otherwise it will default to ``ssh``.
+Using ``mpi`` will likely incur a performance penalty because MPI
+will be running concurrently with GASNet. Running with ``ssh`` is
+recommended, but not all systems support ssh'ing to compute nodes so
+it is not always the default.
+
+
+Using SSH for Job Launch
+------------------------
+
+To launch Omni-Path jobs with SSH, use the following:
+
+.. code-block:: bash
+
+   # Specify ssh spawner
+   export GASNET_OFI_SPAWNER=ssh
+
+   # Specify the nodes to run on (only required when using plain
+   # gasnetrun_ofi outside a Slurm reservation)
+   export GASNET_SSH_SERVERS="nid00001 nid00002 nid00003 ..."
+
+If you receive an error message like:
+
+.. code-block:: printoutput
+
+      *** Failed to start processes on nid00001, possibly due to an inability to establish an ssh connection from login-node without interactive authentication.
+
+This indicates passwordless SSH is not set up. You can try copying
+existing SSH keys or generating new ones with the following:
+
+.. code-block:: bash
+
+      ssh-keygen -t rsa # use default location and empty passphrase
+      cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys
+
+If you see the same error message this may indicate ssh connections
+to compute nodes are not allowed, in which case using the MPI
+spawner may be your only option.
+
+
+Using MPI for Job Launch
+------------------------
+
+To launch Omni-Path jobs with *mpirun*, first make sure that *mpicc* is
+available and that MPI programs launch appropriately with *mpirun*. Then use
+the following. You'll want to make sure that GASNet detects MPI in its
+configuration output.
+
+.. code-block:: bash
+
+   export GASNET_OFI_SPAWNER=mpi
+
+It's worth noting that some configurations do not allow opening the
+network device multiple times from a single process, so this method
+may not be reliable.
+
+
+--------------------
+Verifying Job Launch
+--------------------
+
+Once the above configuration has been done, checking that job
+launching is happening properly is recommended. The following Chapel
```

Comment:
"checking that job launching is happening properly" sounds weird to me. Would "checking that jobs are launching properly" make sense here? Disregard if it doesn't.

Reply:
Changed